service/src/main/java/io/camunda/service/AuditLogServices.java

```diff
@@ -72,8 +81,14 @@ public AuditLogEntity getAuditLog(final String auditLogKey) {
                 .withSecurityContext(
                     securityContextProvider.provideSecurityContext(
                         authentication,
-                        withAuthorization(
-                            AUDIT_LOG_READ_AUTHORIZATION, al -> al.category().name())))
+                        AuthorizationConditions.anyOf(
```

Comment:
🔧 this is pretty much duplicated with line 55, should we store this in a constant/class? or extract into a method?